### DISCUSSION TITLE
Sample dataset?

### DISCUSSION THREAD

**Discussion by deklanw** [RUCAIBox/RecBole#683] [Q&A]
I have a dataset with a particular feature set which I would like to experiment with, but the total number of interactions is very large. Is there a way in RecBole to randomly sample, say, 10% of all users? This way I can experiment with various algorithms and get a rough idea about what might work best for the full dataset.

I see that I can filter by min/max interactions for users/items but that wouldn't quite fit this case

**Comment by johnny12150** [comment]
Maybe you can do it in the preprocessing part of creating`.inter` files?
